packages/pds-ember/package.json

```diff
@@ -40,6 +40,7 @@
   },
   "dependencies": {
     "@ember/render-modifiers": "^1.0.2",
+    "@hashicorp/ember-flight-icons": "^2.0.12",
```

Comment:
@Dhaulagiri I was thinking: is there a risk that in the consumers (eg. Cloud UI) having the same dependency declared in two different places (in Structure `package.json`, and in Cloud `package.json`) this may lead to two packages installed (and imported)? is there a better/safer way to do this?

Reply:
> this may lead to two packages installed

yarn should be smart enough to dedupe these for us